src/js/common/components/authentication/VerifyPrompt.jsx

```diff
@@ -1,31 +1,40 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { handleVerify } from '../../helpers/login-helpers.js';
 
 class VerifyPrompt extends React.Component {
-
   constructor(props) {
     super(props);
-    this.handleVerify = handleVerify;
+    this.handleVerify = this.handleVerify.bind(this);
   }
 
   componentDidMount() {
-		// event for google analytics that we prompted to verify account
     window.dataLayer.push({ event: 'verify-prompt-displayed' });
   }
 
+  handleVerify() {
```

Comment:
Why the move `handleVerify()` out of `login-helpers.js` and into `VerifyPrompt.jsx`?  This function is also used by the `AuthApplicationSelection` component.  The move makes things less DRY.